sql/show.go

```diff
@@ -67,20 +67,25 @@ func (p *planner) ShowIndex(n *parser.ShowIndex) (planNode, error) {
 	v := &valuesNode{columns: []string{"Table", "Name", "Unique", "Seq", "Column"}}
 
 	name := n.Table.Table()
-	for i, index := range desc.Indexes {
-		for j, col := range index.ColumnNames {
-			v.rows = append(v.rows, []parser.Datum{
-				parser.DString(name),
-				parser.DString(desc.Indexes[i].Name),
-				parser.DBool(desc.Indexes[i].Unique),
-				parser.DInt(j + 1),
-				parser.DString(col),
-			})
-		}
+	showOneIndex(desc.PrimaryIndex, name, v)
```

Comment:
i don't think this method is necessary; you can use `for i, index := range append([]structured.IndexDescriptor{*desc.PrimaryIndex}, desc.Indexes...) {`